applications/CompressiblePotentialFlowApplication/python_scripts/apply_far_field_process.py

```diff
@@ -1,6 +1,12 @@
 import KratosMultiphysics
```

Comment:
As a comment aside of this PR, you could think about migrating this process to c++. I think that its efficiency will improve quite a lot.

Reply:
Thanks for the hint! I also heard this from @philbucher several times. It is definately in the todo list to migrate the three processes that we have in python to c++ before passing to 3d.